src/helpers/getCognitoUser.js

```diff
@@ -0,0 +1,13 @@
+import getUserPool from './getUserPool';
+
+export default function getCognitoUser(username, userPoolId, clientId, CognitoUserPool,
+  CognitoUser) {
+  const userPool = getUserPool(userPoolId, clientId, CognitoUserPool);
```

Comment:
This is because the parameter is actually a constructor - ESLint seems fine with this, so I'll keep it as-is for now.